.github/actions/publish-nuget/Update-ManifestVersion.ps1

```diff
@@ -9,7 +9,8 @@ $manifests = Get-ChildItem $WorkDir -File -Recurse -Filter "Manifest.cs" |
     Select-String -List -Pattern '(OrchardCore.Modules.Manifest|OrchardCore.DisplayManagement.Manifest)' |
     Select-Object -ExpandProperty Path
 
-foreach ($manifest in $manifests) {
+foreach ($manifest in $manifests)
+{
```

Comment:
Do we also have an analyzer for this now?

Reply:
VS Code autoformatting fixes this.

Reply:
Could you please do a quick check if there's a built-in or community analyzer for such formatting in PSAnalyzer? I'd be surprised to see a built-in one, because we should've noticed it originally, but for such basics things even a community one would be great.

Reply:
I actually checked the rest of the rules (not enabled by default), now why sure didn't notice them (I was focusing on getting rid of backticks at the time though), but yes, there are: https://github.com/Lombiq/PowerShell-Analyzers/issues/16